locales/it.json

```diff
@@ -0,0 +1,127 @@
+	"sign_out": "Esci",
+	"official_subreddit": "Subreddit Ufficiale",
+	"join_discord": "Chat su Discord",
+	"timelapses": "Timelapse",
+	"source_code": "Codice Sorgente",
+	"donate": "Dona",
+	"home": "Pagina Principale",
+	"copyright": "Dynastic Development 2017",
+	"admin": "Amministratore",
+	"moderator": "Moderatore",
+	"admin_dashboard": "Pannello di Controllo",
+	"admin_actions": "Azioni da Amministratore",
+	"admin_recent_actions": "Azioni Recenti",
+	"admin_modlog": "Log di Moderazione",
+	"admin_users": "Utenti",
+	"admin_reports": "Segnalazioni",
+	"admin_pixels": "Pixel",
+	"global_rank": "Grado globale",
+	"pixel": "Pixel",
+	"pixels": "Pixel",
+	"see_all": "Leggi altro",
+	"coming_soon": "In Arrivo!",
+	"coming_soon_context": "Abbi pazienza mentre lavoriamo su questa funzionalità!",
+	"send": "Invia",
+	"pixel_this_week": "Pixel questa settimana",
+	"pixels_this_week": "Pixel questa settimana",
+	"success": "Successo!",
+	"password_changed": "Password cambiata con successo!",
+	"about_me": "Su di me",
+	"about_them": "Su di loro",
+	"joined": "Iscritto",
+	"x_coordinate": "X",
+	"y_coordinate": "Y",
+	"username": "Nome utente",
+	"default_ban_reason": "Account secondario di ",
+	"or": "o",
+	"password": "Password",
+	"password_repeat": "Password (di nuovo)",
+	"cancel": "Annulla",
+	"error": "Ops!",
+	"save": "Salva",
+	"title": "Titolo",
+	"optional": "(opzionale)",
+	"popout": "Popout",
+	"pixels_placed": "Pixel piazzati",
+	"pixel_placed": "Pixel piazzato",
+	"community_guidelines": "Linee Guida della Comunità",
+	"deactivate": "Disattiva",
+	"account_deactivated": "Account disabilitato",
+	"deactivated_accounts": "Account disabilitati",
+	"last_placed": "Ultimo piazzamento: ",
+	"contact_us": "Contattaci",
+	"message": "Messaggio",
+	"my_account": "Il mio account",
+	"change_password": "Cambia Password",
+	"deactivate_account": "Disabilita Account",
+	"moderator_tools": "Strumenti di Moderazione",
+	"moderator_tools_actions": "Azioni",
+	"go_home": "Vai alla Pagina Principale",
+	"open_issue": "Apri una segnalazione",
+	"chat": "Chat",
+	"leaderboard": "Classifica",
+	"achievements": "Obbiettivi",
+	"active_users": "Utenti Attivi",
+	"oauth_sign_in_with": "Accedi con",
+	"change_password_use_current_password": "Cambia la tua password usando la tua password corrente.",
+	"change_password_irreversable_reminder": "Assicurati di ricordare la nuova password, non puoi annullare questa operazione senza di essa.",
+	"change_password_form_current_password": "Password Corrente",
+	"change_password_form_new_password": "Nuova Password",
+	"change_password_form_new_password_confirm": "Nuova Password (Di nuovo)",
+	"deactivate_account_confirm": "Sei sicuro di voler disabilitare il tuo account?",
+	"deactivate_account_todo": "Prima che te ne vada:",
+	"deactivate_account_irreversable": "Non potrà essere recuperato.",
+	"deactivate_account_what_happens": "I tuoi pixel rimarranno, ma saranno mostrati sotto l'utente \"Account Disabilitato\"."
```

Comment:
I saw it. That's why there's two "I hate myself" commits. They're two missed commas which I didn't notice.